amazon-kinesis-client/src/main/java/software/amazon/kinesis/leases/dynamodb/DynamoDBLeaseSerializer.java

```diff
@@ -268,6 +269,28 @@ protected AttributeValueUpdate putUpdate(AttributeValue attributeValue) {
         return result;
     }
 
+    @Override
+    public Map<String, AttributeValueUpdate> getDynamoUpdateLeaseUpdate(Lease lease,
```

Comment:
why should we update only 2 fields. We can update all the 3 fields without checking for update field why are we doing update for speicific child shards or hash range alone.

Reply:
which three fields. Here, providing the ability to update the fields individually.